### PR TITLE
refactor: TT-295 removal unnecessary feature toggles

### PR DESCRIPTION
**Pull request by kevinjlope**
**Problem**
Currently, there are some unnecessary feature toggles in the UI.
**Solution**
Removed feature toggles and the unit testing associated.

### CODE REVIEW

**Review by JosueOb — APPROVED** [review]
I already tried the local project and had no problem.

**Review by PieritoAlva95 — APPROVED** [review]
Very good work, keep up the good work 👍🏼 💯 🥇

**Review by jase156 — CHANGES_REQUESTED** [review]

File: src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts [hunk 1]
```diff
@@ -176,13 +160,7 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
       return;
     }
     this.entryForm.patchValue({ start_date: newHourEntered });
-    if (this.isFeatureToggleActive) {
-      this.newData.update_last_entry_if_overlap = true;
```
[the remark this answers is not in the document]
Comment (edgardavid2015) [reply src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts:180]: Done.

File: src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts [hunk 2]
```diff
@@ -176,13 +160,7 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
...
-      this.store.dispatch(new entryActions.UpdateEntryRunning({ ...this.newData, ...this.entryForm.value }));
-
-    } else {
-      this.store.dispatch(new entryActions.UpdateCurrentOrLastEntry({ ...this.newData, ...this.entryForm.value }));
```
[the remark this answers is not in the document]
Comment (edgardavid2015) [reply src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts:184]: Done

File: src/app/modules/shared/feature-toggles/feature-toggle-general/feature-toggle-general.service.spec.ts [hunk 3]
```diff
@@ -27,8 +27,7 @@ describe('FeatureToggleGeneralService', () => {
   params.map((param) => {
     it(`isActivated should return a boolean ${param.bool}`, () => {
       const toggleName = FeatureToggle.SWITCH_GROUP;
-      // tslint:disable-next-line: no-shadowed-variable
-      featureManagerService.isToggleEnabledForUser = (toggleName) => of(param.bool);
+      featureManagerService.isToggleEnabledForUser = (toggleNameV) => of(param.bool);
```
[the remark this answers is not in the document]
Comment (edgardavid2015) [reply src/app/modules/shared/feature-toggles/feature-toggle-general/feature-toggle-general.service.spec.ts]: Due to the error "no-shadowed-variable", the variable name changed adding the "V". Now that variable was deleted.

**Review by jcalarcon98 — APPROVED** [review]

**Review by edgardavid2015 — APPROVED** [review]

**Review by jase156 — APPROVED** [review]
Good Job Kevin and Edgar

**Pull request merged**